src/user-utilities.js

```diff
@@ -50,7 +50,21 @@ function createUser(firstName, lastName, email, phoneNumber) {
 
 }
 
+function   manageUsers(user, a) {
+  if ( a== "delete"){
+    return undefined
```

Comment:
We probably don't want to be returning undefined here. That's also the default, so this line isn't doing much.

Reply:
Perhaps no return value, its hard to know if this is supposed to be an error case or what.
Again i think manageUsers is too generic i prefer distinct functions with one purpose
`deleteUser(userId)`
`updateUser(userId, user)`